backend/bamboo/models.py

```diff
@@ -0,0 +1,269 @@
+from datetime import datetime
+
+from sqlalchemy import Integer, String, Text, ForeignKey, Boolean, DateTime, Float
+from sqlalchemy.orm import relationship
+from werkzeug.security import generate_password_hash, check_password_hash
+from sqlalchemy.orm import Mapped, mapped_column
+
+from bamboo.core.extensions import db
+
+
+class User(db.Model):
+    id: Mapped[int] = mapped_column(Integer, primary_key=True)
+    name: Mapped[str] = mapped_column(String)
+    username: Mapped[str] = mapped_column(String)
+    password_hash: Mapped[str] = mapped_column(String)
+    biology: Mapped[str] = mapped_column(String)
```

Comment:
This should be biography? or just bio.
And this could be null so add optional type hint: `bio: Mapped[Optional[str]`, or it would get not null constraint error using sqlite.
All field optional need updates as above.

Reply:
Changed to `bio` and updated optional fields.